config/samples/dspa-simple/dspa_simple.yaml

```diff
@@ -9,10 +9,13 @@ spec:
     cABundle:
       configMapKey: ca.crt
       configMapName: kube-root-ca.crt
+  database:
```

Comment:
Is this change related to this PR? Why we want to disable health checks by default?

Reply:
this snuck in, good catch I'll remove it